### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RyuYongJae60**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@
 Kalkulyator dasturini yaratish
   
 Qoida - Har kim kodni yuklashi mumkin.
-      - Kodni va sharhlar orqali o'z fikrini bildira oladigan har bir kishini yuklang.
+      - Kodni va sharhlar orqali o'z fikrini bildira oladigan har bir kishini yuklang.5454
       - Kod kamida uch kishining ko'rib chiqishi va roziligi bilan asosiy kodga kiritilishi mumkin.
       - Agar tashkilot a'zosi bo'lmagan shaxs kod taklif qilsa, u ko'rib chiqilib, 5 yoki undan ortiq kishining roziligidan so'ng kiritilishi mumkin.
       - Litsenziya GPL 2.0 dan foydalanadi.
```
Comment (Asfanchik) [inline README.md:25]: 123

**Review by taehunlee12 — COMMENTED** [review]
.....

File: README.md (hunk 1 above)
Comment (RyuYongJae60) [inline README.md:25]: qwerqwerqwer

**Review by RyuYongJae60 — COMMENTED** [review]
...

File: README.md (hunk 1 above)
Comment (kyeong98) [reply README.md:25]: abc

**Pull request closed without merging**